daffodil-lib/src/main/scala/org/apache/daffodil/util/RangeBound.scala

```diff
@@ -1,111 +0,0 @@
-/*
```

Comment:
Since RangeBound now depends on DataValue, we had to move it out of daffodil-lib.

For lack of a better place to put it, it is now part of TypeCalculator.scala